Specifications/PropertyBags/for_ImmutableTypeConstructorBasedFactory/when_asking_can_build/for_a_non_immutable_type_with_a_default_constructor.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Dolittle.PropertyBags.Specs.for_ImmutableTypeConstructorBasedFactory.when_asking_can_build
+{
+    using Dolittle.PropertyBags.Specs;
+    using Machine.Specifications;
+
+    [Subject(typeof(ImmutableTypeConstructorBasedFactory),"can_build")]
+    public class for_a_non_immutable_type_with_a_default_constructor
```

Comment:
Love this nested approach..  Something to formalise into documentation; @woksin ?

Reply:
Very nice! I'll forward it to QA-woksin when he's available